tests/Fixer/ClassNotation/FinalClassFixerTest.php

```diff
@@ -75,4 +75,71 @@ public static function provideFixCases(): array
             ['<?php $anonymClass = new class {};'],
         ];
     }
+
+    /**
+     * @dataProvider provideFix80Cases
+     *
+     * @requires PHP 8.0
+     */
+    public function testFix80(string $expected, ?string $input = null): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public static function provideFix80Cases(): array
+    {
+        return [
+            ['<?php #[Entity] class MyEntity {}'],
+            ['<?php use Doctrine\ORM\Mapping as ORM; #[ORM\Entity] class MyEntity {}'],
+            ['<?php use Doctrine\ORM; #[ORM\Mapping\Entity] class MyEntity {}'],
+            ['<?php #[Document] class MyDocument {}'],
+            ['<?php use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM; #[ODM\Document] class MyEntity {}'],
+            ['<?php #[Entity] class MyEntity {}'],
+            ['<?php use Doctrine\ORM\Mapping as ORM; #[ORM\entity] class MyEntity {}'],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFix82Cases
+     *
+     * @requires PHP 8.2
+     */
+    public function testFix82(string $expected, ?string $input = null): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public static function provideFix82Cases(): array
```

Comment:
Can we add some "dirty" cases, not only optimistic ones? Like with comments somewhere in-between etc.

```php
[
    '<?php use Doctrine\ORM; #[ORM\Mapping\Entity] readonly /* ... */ final class MyEntity {}',
    '<?php use Doctrine\ORM; #[ORM\Mapping\Entity] readonly /* ... */ class MyEntity {}',
],
```

Also multiline, maybe comments between attributes and class itself?